service/resource_router_test.go

```diff
@@ -269,7 +269,20 @@ func Test_Service_NewTPRResourceRouter(t *testing.T) {
 			errorMatcher:      nil,
 			resourceRouter:    NewTPRResourceRouter(versionedResources),
 		},
-		// Case 4. Invalid version returns an error.
+		// Case 4. Version 2.0.0 returns all resources except legacy.
```

Comment:
Feel free to ignore, can we add a description field here with the comment contents? And subtests using that description field.

Reply:
I updated the TPR resource router by mistake which doesn't use subtests. It should be frozen until we delete the TPR code.